examples/azure/terraform-azure-nsg-example/main.tf

```diff
@@ -0,0 +1,134 @@
+provider "azurerm" {
+  version = "~>2.20"
+  features {}
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# PIN TERRAFORM VERSION TO >= 0.12
+# The examples have been upgraded to 0.12 syntax
+# ---------------------------------------------------------------------------------------------------------------------
+
+terraform {
+  required_version = ">= 0.12"
+}
+
+# ---------------------------------------------------------------------------------------------------------------------
+# DEPLOY A RESOURCE GROUP
+# See test/terraform_azure_example_test.go for how to write automated tests for this code.
+# ---------------------------------------------------------------------------------------------------------------------
+
+resource "azurerm_resource_group" "main" {
```

Comment:
suggest changing resource name to vnet_rg

Reply:
Updated resource name to match (nsg_rg)